neutone_sdk/utils.py

```diff
@@ -157,13 +157,18 @@ def save_neutone_model(
             loaded_model, loaded_model.get_preserved_attributes()
         )
         log.info("Testing methods used by the VST...")
-        loaded_model.calc_min_delay_samples()
         loaded_model.set_daw_sample_rate_and_buffer_size(48000, 512)
+        loaded_model.calc_buffering_delay_samples()
```

Comment:
We don't really need these two calls here since we call these methods underneath anyway.